src/management/OrganizationsManager.js

```diff
@@ -0,0 +1,816 @@
+ * </caption>
+ *
+ * // Pagination settings.
+ * var params = {
+ *   per_page: 10,
+ *   page: 0
+ * };
+ *
+ * management.organizations.getAll(params, function (err, organizations) {
+ *   console.log(organizations.length);
+ * });
+ *
+ * @param   {Object}    [params]          Organizations parameters.
+ * @param   {Number}    [params.per_page] Number of results per page.
+ * @param   {Number}    [params.page]     Page number, zero indexed.
+ * @param   {Function}  [cb]              Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+utils.wrapPropertyMethod(OrganizationsManager, 'getAll', 'organizations.getAll');
+
+/**
+ * Get an Auth0 organization.
+ *
+ * @method    get
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * management.organizations.get({ id: ORGANIZATION_ID }, function (err, role) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ *
+ *   console.log(organization);
+ * });
+ *
+ * @param   {Object}    params        Organization parameters.
+ * @param   {String}    params.id     Organization ID.
+ * @param   {Function}  [cb]          Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+utils.wrapPropertyMethod(OrganizationsManager, 'get', 'organizations.get');
+
+/**
+ * Update an existing organization.
+ *
+ * @method    update
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var data = { name: 'New name' };
+ * var params = { id: ORGANIZATION_ID };
+ *
+ * // Using auth0 instance.
+ * management.updateOrganization(params, data, function (err, organization) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ *
+ *   console.log(organization.name);  // 'New name'
+ * });
+ *
+ * // Using the organizations manager directly.
+ * management.organizations.update(params, data, function (err, organization) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ *
+ *   console.log(organization.name);  // 'New name'
+ * });
+ *
+ * @param   {Object}    params        Organization parameters.
+ * @param   {String}    params.id     Organization ID.
+ * @param   {Object}    data          Updated organization data.
+ * @param   {Function}  [cb]          Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+utils.wrapPropertyMethod(OrganizationsManager, 'update', 'organizations.patch');
+
+/**
+ * Delete an existing organization.
+ *
+ * @method    delete
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * management.organizations.delete({ id: ORGANIZATION_ID }, function (err) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ *
+ *   // Organization deleted.
+ * });
+ *
+ * @param   {Object}    params        Organization parameters.
+ * @param   {String}    params.id     Organization ID.
+ * @param   {Function}  [cb]          Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+utils.wrapPropertyMethod(OrganizationsManager, 'delete', 'organizations.delete');
```

Comment:
I am not sure how this utils work, I will just assume it works. (applies to all utils calls)

Reply:
It's not very intuitive 😅

Reply:
It works :P